geometry_msgs/Point position 
---
p3/JointAngles angles
